mbuild/compound.py

```diff
@@ -321,6 +325,7 @@ def __init__(self, subcompounds=None, name=None, pos=None, charge=0.0,
         self._check_if_contains_rigid_bodies = False
 
         self.box = box
+        self.element = element
```

Comment:
Just out of curiosity, will this line actually call the setter method for element or does it just create a `element` variable for Compound? 
 If it's the second case, should this become `self._element = get_by_symbol(element)`?

Reply:
Pretty sure it calls the setter. At least that has always been my experience.